revolv/templates/base/partials/impact_statistics.html

```diff
@@ -0,0 +1,80 @@
+{% load staticfiles %}
+{% load humanize %}
+{% load revolv_cms_tags %}
+
+<div class="large-12 columns">
+	<div class="row statistics-container">
+		<div class="row">
+			<p class="contribution-text">{{ settings.revolv_cms.DashboardSettings.impact_statistics_header_text }}</p>
+		</div>
+		<div class="row statistics-row">
+			<div class="medium-6 columns statistic col-1">
+			</div>
+			<div class="medium-6 columns statistic col-2">
+			</div>
+		</div>
+		<div class="row statistics-row">
+			<div class="medium-6 columns statistic col-1">
+				<div class="medium-4 small-5 columns icon-container">
+					<img class="icon repayments-icon" src="{% static "images/coin.svg" %}" />
+				</div>
+				<div class="medium-8 small-7 columns">
+					<div class="text-container repayments">
+						<p class="top-description description">earned</p>
+						<p class="emphasized-text repayments">${{ statistics.repayments|floatformat:2|intcomma }}</p>
+						<p class="bottom-description description">in repayments</p>
+					</div>
+				</div>
+			</div>
+			<div class="medium-6 columns statistic col-2">
+				<div class="medium-4 small-5 columns icon-container">
+					<img class="icon projects-donated-to-icon" src="{% static "images/house.svg" %}" />
+				</div>
+				<div class="medium-8 small-7 columns">
+					<div class="text-container number-of-projects">
+						<p class="description">contributed to</p>
+						<p class="emphasized-text number-of-projects">{{ statistics.project_count }} project{{ statistics.project_count|pluralize }}</p>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="row statistics-row">
+			<div class="medium-6 columns statistic col-1">
+			</div>
+			<div class="medium-6 columns statistic col-2">
+				<div class="medium-4 small-5 columns icon-container">
+					<img class="icon last-icon" src="{% static "images/revolv-man.svg" %}" />
+				</div>
+				<div class="medium-8 small-7 columns">
+					<div class="text-container save-world">
+						<p class="description last-icon">Helped us save</p>
```

Comment:
I think we could make this CMS editable too! Also, I don't think it needs to be broken into two divs